src/main/java/com/launchacademy/reviews/controllers/UsersAPIController.java

```diff
@@ -1,9 +1,25 @@
 package com.launchacademy.reviews.controllers;
 
+import com.launchacademy.reviews.models.User;
+import com.launchacademy.reviews.repositories.UserRepository;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 @RequestMapping("/api/v1/users")
 public class UsersAPIController {
+  private final UserRepository userRepository;
+
+  @Autowired
+  public UsersAPIController(UserRepository userRepository){
+    this.userRepository = userRepository;
+  }
+
+  @GetMapping
+  public Iterable<User> getAllUsers(){
+    return userRepository.findAll();
+  }
```

Comment:
These all look awesome! Just make sure you're only creating API endpoints you'll end up using -- for example, we don't want to expose User data unless we need to.

As a future refactor, this could also be a great place to use a Serializer, to protect user information and only show what's needed for the page!